### DISCUSSION TITLE
Chlorinator Relay Needed if controlled from RS485?

### DISCUSSION THREAD

**Discussion by jamestalmage** [tagyoureit/nodejs-poolController#737] [Nixie] [answered]
The documentation calls for the IntelliChlor to be hooked to the relay for the pump.... (Nixie docs, and Pentair docs both)

The documentation also says no relay should be used for RS-485 controlled pumps (Nixie docs... Didn't look up the easytouch docs).

Basically, there's ambiguity... Is it best practice to add a relay for the IntelliChlor if I'm not using a relay for the pump?

**Comment by rock-crusher** [comment]
Yes, the SCG must be connected via the pump filter relay.
The relay hat on your PI will accommodate the SCG.

**Reply by jamestalmage** [reply]
To be clear... I'm not planning on a pump relay. I'm controlling the pump via RS-485.
So I'm adding one specifically for the SWCG, and just labelling it as pump-filter in the software?

> The relay hat on your PI will accommodate the SCG.

Meaning no need to use an external 3HP pump relay? Because we're not starting a motor, back EMF isn't a concern?

**Reply by rock-crusher** [reply]
Well, if I’m not mistaken the Pentair power supply for the IC40 is only rated 2amp on the input. That said, the relay hat should be capable of the load. But… I personally prefer to use the hat relays only to engage a contactor for such loads just to prevent the possible damage to the hat.

**Comment by jamestalmage** [comment]
Trying to understand... Why is it acceptable to rely on RS-485 to communicate a stop signal to a pump, but not an SWCG?

It seems to me that the whole point of the original setup (putting SWCG on the SAME relay as the pump) was that it guaranteed your pump was running when the SWCG had power... With the pump on constant power, we're back to relying on the flow switch working, and the automation system not calling for it at improper times...

Is it just a relay-reliability vs RS-485 reliability judgment call? (And if that's the case, then it seems like it should have been easy for Pentair to solve by requiring a heartbeat signal from the automation system to keep the SWCG running)

**Reply by rstrouse** [reply, marked as answer]
Just a little bit of a backgrounder on the differences between the RS485 comms for pumps and SWCG.

The chlorinator messages operate based on watchdog messages.  These are sent at an interval to disable the control panel and set the current level.  If the chlorinator does not get a message every 15 or so seconds it will then re-enable the control panel.  RS485 pumps and heaters operate exactly the same way.

However, where this starts to diverge is typically for pumps and heaters their default operation mode is off when you simply stop talking to it.  Sadly, this is not the case with a chlorinator.  It reverts back to whatever its last setting was and re-enables the controls.  So in the case of a pump, comms failure leads to the pump not running.  The same is true for a heater.   In the case of a chlorinator it will merrily continue to generate chlorine if power is still applied.  

As a secondary, and the most common failure point, chlorinators utilize a flow switch but this is primarily to protect against the scenario where a pump fails and is simply no longer moving water or it cavitates and is no longer moving water.  Newer pump versions will eventually shut off but that is because of the thermal protection or runaway current draw.  This can go on for hours until the pump gives up the ghost.

So the reason, they put pumps and chlorinators on the same relay is simply to protect against a failure rate of stupidity.  Where the user kills the power to the pump but keeps the automation active.  If the cell fails with the flow switch closed then this is not a good situation and will fill up the plumbing with hydrogen gas over time.  

Essentially, all of this is to reduce the risk for multiple failure points because the failure can be catastrophic.  If the failure simply resulted in equipment damage then it wouldn't be so important.

**Reply by jamestalmage** [reply]
Understood.

Is an external 3hp pump relay required/recommended for Intellichlor? Hat relays won't do?

**Reply by rstrouse** [reply]
Double check the AC side of the transformer.  If the draw is less than 4A 120vac then the relay and its connectors are rated for that output.

**Reply by rock-crusher** [reply]
>  Is an external 3hp pump relay required/recommended for Intellichlor? Hat relays won't do?

Well, the external relay may in fact be needed if you choose to follow Pentair’s recommendations.

Here is another bit of info:
The Pentair instructions also infer that the SCG power supply is to be connected to the same power source as the pump, that way if the breaker was turned off to the pump for service, or other reasons, so would the power to the SCG. 

This would suggest that an external 2 pole relay (such as the 3hp pump relay or similar) would then be required (even though the pump is controlled via RS485 and connected directly to a circuit breaker) for the SCG (wired at 240v) and connected to the same circuit breaker as the pump.

James, you have brought up many of the same questions others have in the past regarding this topic, it reminded me of this discussion. You will have to read through it a little until the SCG topic comes up.
Here is the link: [](url)https://github.com/tagyoureit/nodejs-poolController/discussions/484

**Comment by truvec** [comment]
I have a Hayward T15 and I'm looking into this as well. Definitely, having it on a relay makes sense. The part I'm still looking at is the type of relay. 

IMO: It's better to run on 120v so that it can use a single pole relay to remove power. 

Initially, I was stuck thinking about running on 220v as if it was somehow superior. Well, since "watts are watts" and this device does not use a lot of them(relatively speaking). 

> IntelliChlor SCG Model IC20, IC40 and IC60
> Input: 95-130 VAC, 50/60 Hz, 220 Watts (2 AMP) or 220-240 VAC, 50/60 Hz, 220 Watts (1 AMP). Output: 22-39 VDC @ 7.5 AMPS maximum from the Power Center.
> 

My plan is to run it 120v with a relay hat. If the relay fails I'll fall back to a beefier relay.

**Reply by rock-crusher** [reply]
Hi truvec,

I am only familiar with the Pentair SCG and as stated above it is to be powered by the same source as the pump. That said if your pump is 240v and connected to a 2pole GFCI circuit breaker you would then need a 2pole relay connected to the same breaker on the relay line side and the SCG power supply (wired for 240v) connected to the load side of the relay. The 2pole relay would then be connected to and controlled by Nixie’s filter relay circuit. 

Wired this way, when the pump breaker is turned off for any reason the SCG will shut down as well.

This is an additional safety feature Pentair requires.

![Warning](https://user-images.githubusercontent.com/94593954/234383021-9e583c70-c9f2-4728-a8f3-14007716afa9.JPG)

![IntelliChlor SCG](https://user-images.githubusercontent.com/94593954/234383081-a58532b3-b1ce-43d2-be57-aadb66166048.JPG)

**Reply by truvec** [reply]
I get that you are promoting that the circuit that powers the pump should also power the SWG. Seems good.

But saying that both the pump and the SWG need to be 240v(or double pole) is not accurate.

My plan is to hook the pump(240) directly to load center and rely on the nixie filter relay to control the SWG 120V.  Given most onboard hats are single pole its best to be able to just cut the one leg to shut down.

**Reply by jamestalmage** [reply]
@truvec - The point here is making sure that they are on the same breaker. Just wiring both of them to the same two-pole breaker at 240v seems the most straightforward. You could essentially center-tap the pump breaker so the SWG is only wired across one leg... But that's going to confuse someone else who's servicing it down the line.... And it means there's 120v in your PI stack. What boards are either side of your relay board in the stack?

I'm thoroughly convinced on the external relay now... Small price to pay for the added peace of mind and simplicity it brings.

**Reply by truvec** [reply]
I definitely had concerns and thoughts about running HV in the stack. But I decided to run the light transformer through the hat. 

I'm going to run 4 lights, 3 actuators, 3 temps, pressure, orp, ph, conductivity, two totalizers and 1 valve. 

I like the idea of keeping the stack LV dc and use separate relays for the HV switching. I'm going to need a bigger box. 

Servicing a nixie box? By a pool tech? LOL. 

I'm thinking about relocating the two relays for the Light transformer and SWG to my load center. I'll put two relays in there and drive them from the hat.

I still stand by my point about the single pole relay. If someone is on a budget (or all that's on hand) it's totally fine to control the SWG 120 single pole. My first automated pump controller was an SSR driven by a Z wave switch(all taken from the parts bin).

Thanks

**Reply by rock-crusher** [reply]
Hi again truvec,

Absolutely you can wire it up the way you propose, but it may cause nuisance tripping of the 2 pole GFCI breaker supplying your pump. Those breakers can be exceptionally sensitive to unbalanced current.  

The need for the 2 pole relay was my way of keeping it simple and compliant, tapping one leg of a 2 pole dedicated circuit to supply power for other equipment isn’t the best way to accomplish this goal, but yes, it will work.

**Reply by jamestalmage** [reply]
2-pole GFCIs allow for unbalanced current. They monitor all three legs (both phases and the neutral).

**Reply by rock-crusher** [reply]
Ah yes, your absolutely right James!  

Thanks for bringing that up! 

I’ve never used a 2pole GFCI in that manner. I completely forgot about the neutral terminal tucked up in there.  😜
